src/test/scala/com/snowflake/snowpark/UtilsSuite.scala

```diff
@@ -672,6 +673,37 @@ class UtilsSuite extends SNTestBase {
     assert(Utils.quoteForOption("FALSE").equals("FALSE"))
     assert(Utils.quoteForOption("abc").equals("'abc'"))
   }
+
+  test("Scala and Json format transformation") {
+    val javaHashMap = new util.HashMap[String, String]() {
+      {
+        put("one", "1")
+        put("two", "2")
+        put("three", "3")
+      }
+    }
+    val map = Map(
+      "nullKey" -> null,
+      "integerKey" -> 42,
+      "shortKey" -> 123.toShort,
+      "longKey" -> 1234567890L,
+      "byteKey" -> 123.toByte,
+      "doubleKey" -> 3.1415926,
+      "floatKey" -> 3.14F,
+      "boolKey" -> false,
+      "javaListKey" -> new util.ArrayList[String](util.Arrays.asList("a", "b")),
+      "javaMapKey" -> javaHashMap,
+      "seqKey" -> Seq(1, 2, 3),
+      "arrayKey" -> Array(1, 2, 3),
+      "seqOfStringKey" -> Seq("1", "2", "3"),
+      "stringKey" -> "stringValue",
+      "nestedMap" -> Map("insideKey" -> "stringValue", "insideList" -> Seq(1, 2, 3)),
+      "nestedList" -> Seq(1, Map("nestedKey" -> "nestedValue"), Array(1, 2, 3)))
+    val jsonString = Utils.mapToJson(map)
```

Comment:
Please add a hard code string comparison here.
```scala
assert(jsonString == "<the expected output>"
```
Otherwise, we don't know if the output of `mapToJson` is correct.